cfme/tests/infrastructure/test_vm_analysis.py

```diff
@@ -390,6 +408,8 @@ def _scan_test(provider_crud, vm, os, fs_type, soft_assert):
         soft_assert("os-linux_fedora.png" in details_os_icon)
         soft_assert("linux_fedora" in quadicon_os_icon)
 
+pytestmark = [pytest.mark.parametrize("rest", ["using_ui", "rest_detail", "rest_collection"])]
```

Comment:
Using parametrize to create new test cases is a little dirty, and it'll probably need to be undone (and be easier to read and debug) when _scan_test is broken up.